src/pages/_LoginPage.js

```diff
@@ -38,6 +46,37 @@ const LoginPage = () => {
 						</Link>
 					</SigninForm.Frame>
 
+					<SigninForm.ReCaptchaHeader>
```

Comment:
That is impressive to include the ReCatcha!! I will check it out and test if I am human